extension/js/common/api/email-provider/sendable-msg.ts

```diff
@@ -83,9 +83,16 @@ export class SendableMsg {
       }
     }
     this.headers.Subject = this.subject;
-    if (this.type === 'pgpMimeSigned' && this.sign) {
+    if (this.type === 'smimePlain' && this.body['encrypted/buf']) {
+      return await Mime.encodePlain(this.body['encrypted/buf'], this.headers);
+    } else if (this.type === 'pgpMimeSigned' && this.sign) {
       return await Mime.encodePgpMimeSigned(this.body, this.headers, this.atts, this.sign);
     } else {
+      // encrypted/buf is a Buf instance that is converted to one part text/plain
+      // message
+      if (this.body['encrypted/buf']) {
```

Comment:
Here somewhat confusing - is any usecase still currently using this option - encrypted/buf, if the `smime` variant is handled above? Eg as is, will this branch ever run?

@wiktor-k